backend/canvas_scopes.py

```diff
@@ -0,0 +1,7 @@
+DEFAUlT_CANVAS_SCOPES = [
```

Comment:
maybe we should put those values as default setting in .env.sample file?

Reply:
Yeah, I'd probably put these 3 in the config.

Reply:
Done. https://github.com/tl-its-umich-edu/canvas-app-explorer/pull/245/commits/9be2ef56d714325811b73dd16f24fe97fda9c68b

Reply:
Verified.